uberfire-nio2-backport/uberfire-nio2-api/src/main/java/org/uberfire/java/nio/file/api/FileSystemUtils.java

```diff
@@ -37,6 +38,10 @@ public static boolean isOpenShiftSupported() {
         return "true".equals(getConfigProps().getProperty(CFG_KIE_CONTROLLER_OCP_ENABLED, "false"));
     }
 
+    public static boolean isMonitoringOnOpenShiftSupported() {
+        return FileSystemUtils.isOpenShiftSupported() && ServerType.isMonitoring();
+    }
+
     public static boolean isK8SFileSystemProviderAsDefault(FileSystemProvider provider) {
         return isOpenShiftSupported() && K8S_FS_SCHEME.equals(provider.getScheme());
```

Comment:
isOpenShiftSupported() && K8S_FS_SCHEME.equals(provider.getScheme()) && ServerType.isMonitoring();

Reply:
@adrielparedes Are we planning to make the suggested change before merge, or you have other thoughts?